flows/vision-zero/prefect_bootstrap/start_agent.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 
+/etc/init.d/ssh start
```

Comment:
need to start an ssh server so we can self-serve as our own bastion

Reply:
This is cool for sure. Could an alternative to this be creating specific DB connect utils for local and production use?  An on/off switch of sorts for the manner of connection (one that tunnels and one that does not)? Just a thought for the next iteration.